### DISCUSSION TITLE
Autogluon predict in kaggle

### DISCUSSION THREAD

**Discussion by Mon1ster** [autogluon/autogluon#3629] [Q&A]
After training the autogluon model, we get the "ag-20231028_172126" file. But there is a file in it which is "__version__", and the file name starts with an underscore "__", so it can't be imported into kaggle, and it can't be predicted in kaggle any more, what should we do about it?

**Comment by samvelkoch** [comment]
Step 1 - Train model(s) in Notebook 1, predictor.save(), Notebook 1 - save with output
Step 2 - Create Inference Notebook 2. 'Add Data' - 'Your notebooks' - Notebook 1 
Step 3 - predictor = TabularPredictor.load("/kaggle/input/Notebook 1/AutogluonModels/path")
Step 4 - predictor.predict()

done

* a bit trickier for multi-label  from https://auto.gluon.ai/stable/tutorials/tabular/advanced/tabular-multilabel.html 

instead of 

```
    def get_predictor(self, label):
        """ Returns TabularPredictor which is used to predict this label. """
        predictor = self.predictors[label]      
        if isinstance(predictor, str):
             return TabularPredictor.load(path=predictor)
         return predictor
```

use

```
    def get_predictor(self, label):
        """ Returns TabularPredictor which is used to predict this label. """
        predictor = self.predictors[label]        
        if isinstance(predictor, str):
            path = '/kaggle/input/Notebook 1/' + predictor  # changes done here
        return TabularPredictor.load(path=path)
```
